Reminder-App/Reminder-App/ReminderDetailViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ReminderDetailViewController.swift
+//  Reminder-App
+//
+//  Created by BrainX Technologies 11 on 15/07/2021.
+//
+
+import UIKit
+
+class ReminderDetailViewController: UITableViewController {
+    
+    enum ReminderRow: Int, CaseIterable {
+        case title
+        case date
+        case time
+        case notes
+        
+        func displayText(for reminder: Reminder?) -> String? {
+            switch self {
+            case .title:
+                return reminder?.title
+            case .date:
+                return reminder?.dueDate.description
+            case .time:
+                return reminder?.dueDate.description
+            case .notes:
+                return reminder?.notes
+            }
+        }
+        
+        var cellImage: UIImage? {
+            switch self {
+            case .title:
+                return nil
+            case .date:
+                return UIImage(systemName: "calendar.circle")
```

Comment:
We always define our strings, colors, fonts, constants and image names in globals. So create a group (folder) named globals and store these constants in swift file..